Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityCameraProfileInspector.cs

```diff
@@ -1,20 +1,17 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information. 
 
-using Microsoft.MixedReality.Toolkit.Utilities;
-using Microsoft.MixedReality.Toolkit.Utilities.Editor;
-using UnityEngine;
-using UnityEditor;
 using Microsoft.MixedReality.Toolkit.CameraSystem;
+using UnityEditor;
+using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.Editor
 {
     [CustomEditor(typeof(MixedRealityCameraProfile))]
-    public class MixedRealityCameraProfileInspector : BaseMixedRealityToolkitConfigurationProfileInspector
+    internal class MixedRealityCameraProfileInspector : BaseDataProviderServiceInspector
```

Comment:
Should we leave these public, in case these classes to be extended if devs extend the MixedRealityCameraSystem/Profile?

Reply:
eh...these aren't exactly the kind of classes we want people to extend or expect to. But I'll switch to public so at least it's possible I guess